src/client/components/Label/Tags/Label.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './label.style.css';
+
+const Labels = ({ text }) => <label className="button">{text}</label>;
```

Comment:
Why is this plural if it is only one label inside? Do you have plans on adding more inside? Or did you accidentally name it like this?